### ISSUE TITLE
Latency offset field is limited

### ISSUE THREAD

**Issue by cjvth** [lxqt/pavucontrol-qt#258]
##### Expected Behavior

If you connect bluetooth headphones and try to change the latency offset, you have no limits. Just like in original pavucontrol

##### Current Behavior

The number box accepts values from 0 to 99

##### Possible Solution

Remove the box max and min value or increase them

##### Steps to Reproduce (for bugs)

1. Connect bluetooth headphones
2. Open pavucontrol-qt 
3. Go to Output devices > the headphones > Show additional settings > Latency offset
4. Try to set value that is higher than 100 or less than 0

##### Context

I needed a to set a negative offset after some changes in Pipewire.

##### System Information

* Distribution & Version: EndeavourOS
* Kernel: 6.2.12
* Qt Version: 5.15.9
* libqtxdg Version: -
* lxqt-build-tools Version: -
* Package version: 1.3.0

**Comment by RJVB** [comment]
This patch gives the offset spinners the same range as they have in `pavucontrol`:

```
diff --git a/src/devicewidget.ui b/src/devicewidget.ui
index 3b5f781..5b8ca1e 100644
--- a/src/devicewidget.ui
+++ b/src/devicewidget.ui
@@ -244,6 +244,12 @@
            <property name="suffix">
             <string> ms</string>
            </property>
+           <property name="minimum">
+            <number>-2000</number>
+           </property>
+           <property name="maximum">
+            <number>5000</number>
+           </property>
           </widget>
          </item>
         </layout>
```

**Comment by tsujan** [comment]
I know nothing of the code of PulseAudio, but if that spin-box shows something in milliseconds, these lines seem suspicious to me:

https://github.com/lxqt/pavucontrol-qt/blob/6502a29663b7561faeadb768e751121c146ee4a0/src/devicewidget.cc#L150

https://github.com/lxqt/pavucontrol-qt/blob/6502a29663b7561faeadb768e751121c146ee4a0/src/devicewidget.cc#L177

@palinek?

**Comment by RJVB** [comment]
On Tuesday July 30 2024 16:27:59 tsujan wrote:

With my patch, pavucontrol-qt latency spinboxes show the values I configured via pavucontrol (gtk).

>I know nothing of the code of PulseAudio, but if that spin-box shows something in milliseconds, these lines seem suspicious to me:

The spinboxes show milliseconds indeed. Pulseaudio uses its own type for latency offsets: `pa_usec_t`. Usec must stand for microseconds.

The interesting thing here is that it's an unsigned type and so the question is how negative values (configured in the spinners) are handled.

**Comment by tsujan** [comment]
> Pulseaudio uses its own type for latency offsets: `pa_usec_t`.

In that case, those lines are correct. Thanks!

**Comment by tsujan** [comment]
@RJVB 
Would you make a PR for the limits of that spin-box? The current limits are practically useless.

**Comment by RJVB** [comment]
>Would you make a PR for the limits of that spin-box? The current limits are practically useless.

I could, but it's not high on my bucket list.

Plus I do not yet have Qt6 on any of my systems so "formally" I can only do PRs against the non-existing Qt5 branch ;)

**Comment by tsujan** [comment]
Done in https://github.com/lxqt/pavucontrol-qt/pull/280

**Comment by RJVB** [comment]
On Thursday August 01 2024 00:07:23 tsujan wrote:
>Closed #258 as completed via 3cf4a5ef7b4d297d8f7284f73039f6df77daef8f.

Thanks. I meant to post a LGTM on the PR but got called away, sorry.
